slices/fonts-ubuntu.yaml

```
@@ -0,0 +1,55 @@
+package: fonts-ubuntu
+
+essential:
+  - fonts-ubuntu_copyright
+
+slices:
+  all:
+    essential:
+      - fonts-ubuntu_ubuntu
+      - fonts-ubuntu_ubuntu-mono
+      - fonts-ubuntu_ubuntu-sans
+      - fonts-ubuntu_ubuntu-sans-mono
+
+  config:
+    contents:
+      /etc/fonts/conf.d/71-ubuntulegacy.conf:
```

Should we add a wildcard here? Not sure if this could be prone to change. Also applies to line below.
```suggestion
      /etc/fonts/conf.d/*-ubuntulegacy.conf:
```

---

I'd keep it. Just to be cautious of potential conflicts.